src/standard/builtin.mli

```diff
@@ -96,7 +96,7 @@ type _ t +=
 type _ t +=
   | In_interval of bool * bool
   (** [In_interval (b1, b2): Int -> Int -> Int -> Prop]:
```

Comment:
You should also update the type schema on this line.

Reply:
Done